React-Native/en/3.3.3.md

```diff
@@ -0,0 +1,7 @@
+# [Success Criterion 3.3.3 - Error Suggestion](https://www.w3.org/WAI/WCAG21/Understanding/error-suggestion.html)
+
+In React Native there is no standard to suggest solutions. You can use the [Text](https://reactnative.dev/docs/text) component to display an error solution. Whenever an error occurs and a solution is shown, users of assistive technologies should also be able to identify this.
```

Comment:
Are there any packages for showing suggestions on input fields?

Reply:
Same answer here as for [3.3.1](https://github.com/appt-org/accessibility-code-examples/pull/3#discussion_r940046863). Doesn't really add to what we already describe here in my opinion.

Reply:
Use text of 3.3.1 but replace 'helper text' with 'suggestion for resolving the error' or something like that?